tools/calendar/delete-events/delete-events.go

```diff
@@ -26,49 +34,114 @@ func main() {
 	if serviceEmail == "" || privateKey == "" {
 		log.Fatal("FLEET_TEST_GOOGLE_CALENDAR_SERVICE_EMAIL and FLEET_TEST_GOOGLE_CALENDAR_PRIVATE_KEY must be set")
 	}
-	userEmail := flag.String("user", "", "User email to impersonate")
+	userEmails := flag.String("users", "", "Comma-separated list of user emails to impersonate")
```

Comment:
@getvictor is this related to the backoff ?

Reply:
These are scripts to use during load testing. I've updated them.